lib/text/ui_text_displayer.js

```diff
@@ -269,7 +278,9 @@ shaka.text.UITextDisplayer = class {
     }
     if (updateDOM) {
       for (const cueElement of toUproot) {
```

Comment:
Technically, now that `toUproot` can contain both cue elements and region elements, this variable should no longer be called `cueElement`. Perhaps just `element`?

Reply:
SGTM

Reply:
Done.